docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,199 @@
+
+Installation
+============
+
+For the first step of our journey, let's install MCDR
+
+.. note::
+
+    We assume you have some really basic knowledge of Python and pip.
+
+    If you have not, Google is your friend
+
+Requirements
+------------
+
+.. image:: https://img.shields.io/pypi/pyversions/mcdreforged.svg
+   :alt: Python version
+
+.. image:: https://img.shields.io/pypi/v/mcdreforged.svg
+   :alt: PyPI version
+
+MCDR is written and runs in Python 3. Make sure you have Python 3.8 or later installed
+
+The detailed Python version requirements are as shown in the table below
+
+.. list-table::
+   :header-rows: 1
+
+   * - MCDR version
+     - Python requirement
+   * - < 2.10
+     - >= 3.6
+   * - >= 2.10
+     - >= 3.8
+
+Install using pip...?
+----------------------
+
+MCDR is available in `PyPI <https://pypi.org/project/mcdreforged>`__, which means it can be installed via the ``pip`` command:
+
+.. code-block:: bash
+
+    pip install mcdreforged
+
+Externally managed environment
+------------------------------
```

Comment:
A summary table for all described installing methods (pip, pipx, venv, docker, system package) will be nice to have

Example pros and cons:

| method          | pros                        | cons                                  |
|-----------------|-------------------------------|---------------------------------------|
| pip             | Native, always available      | May affect global packages with root  |
| pipx            | Easy usage                 | Different command set from pip        |
| venv            | Native support                | Requires manual environment activation       |
| docker          | Reliable across environments       | Needs more dependencies and disk space   |
| system package  | none                          | Not recommended                       |